babel.cfg

```diff
@@ -3,11 +3,11 @@
 [jinja2: templates/**.html]
```

Comment:
Don't we mind the `jinja2` keyword though? :p

Reply:
We went from Jinja2==2.10.3 to 3.1.2, thus I was joking about it not being Jinja2 anymore but Jinja3 :p
Please don't mind me